nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -0,0 +1,61 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Optional
+
+import openvino.runtime as ov
+
+from nncf.data import Dataset
+from nncf.quantization.telemetry_extractors import CompressionStartedWithQuantizeApi
+from nncf.parameters import convert_ignored_scope_to_list
+from nncf.parameters import IgnoredScope
+from nncf.parameters import ModelType
+from nncf.parameters import TargetDevice
+from nncf.common.quantization.structs import QuantizationPreset
+from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantization
+from nncf.quantization.algorithms.post_training.algorithm  import PostTrainingQuantizationParameters
+from nncf.telemetry import tracked_function
+from nncf.telemetry.events import NNCF_OV_CATEGORY
+
+
+@tracked_function(NNCF_OV_CATEGORY, [CompressionStartedWithQuantizeApi(), "target_device", "preset"])
+def quantize_impl(model: ov.Model,
+                  calibration_dataset: Dataset,
+                  preset: QuantizationPreset,
+                  target_device: TargetDevice,
+                  subset_size: int,
+                  fast_bias_correction: bool,
+                  model_type: Optional[ModelType] = None,
+                  ignored_scope: Optional[IgnoredScope] = None) -> ov.Model:
+    """
+    Implementation of the `quantize()` method for the OpenVINO backend via the Model API.
```

Comment:
`Model API` is something that exists in OMZ. I would call it like `...for the OpenVINO backend via the OpenVINO Python API.`